Dockerfile

```diff
@@ -32,6 +32,12 @@ RUN dpkg --add-architecture i386 && \
     python-wxgtk3.0 \
     && apt-get clean
 
+
+# Allows us to invalidate cache if those repos update.
+# Intentionally not pinning for dev velocity.
+ADD https://github.com/kollivier/python-for-android/archive/webview_plus.zip p4a.zip
+ADD https://github.com/kollivier/pyeverywhere/archive/dev.zip pew.zip
+
 # install python dependencies
 RUN curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && \
   python get-pip.py && \
```

Comment:
The `pip install` commands appear to still pull from the git repos for pew and p4a. When I switch them over to use the zip files instead, I get the "can't find native/android/init.sh" problem.

Reply:
They do indeed. I figured downloading the 1.8 meg files twice won't be that big an issue. Based on [this issue I found](https://github.com/pypa/pip/issues/6586), the problem is that `pip` supports installing from GH/a git repo if you prefix with `+git`, and you can install from an archive, but not a git repo in an archive (`git+/pew.zip` doesn't work).

Thought about unzipping, then installing from the unpacked archive, but the unzipped archive name varies by branch and... I figured this might be simple enough.